client/src/components/Modal/Modal.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./Modal.css";
+
+const Modal = props => {
+  return (
+    <div
+      className="Modal"
+      style={{
+        transform: props.showModal ? "translateY(0vh)" : "translateY(-100vh)",
+        opacity: props.showModal ? "1" : "0"
+      }}
+    >
```

Comment:
This is a good way of handling this. Another is to create 'state' classes like `.is-visible` or `.is-hidden` and then toggle adding or removing those classes.

Reply:
You don't have to go with that

Reply:
Yea, whoever in 33 was working on it, implemented that functionality, it's kinda nice since it allows for a nice sliding transition